autoload/lsp/omni.vim

```diff
@@ -140,11 +140,23 @@ function! s:get_completion_result(data) abort
         let l:incomplete = 0
     endif
 
+    let l:current_line = strpart(getline('.'), 0, col('.') - 1)
+    let l:start_pos = match(l:current_line, '\k\+$')
```

Comment:
we might need to make the regex configurable in the future but for now is ok. When someone complains we can make it configure per filetype.